depends/packages/miniupnpc.mk

```diff
@@ -3,17 +3,18 @@ $(package)_version=2.2.2
 $(package)_download_path=https://miniupnp.tuxfamily.org/files/
 $(package)_file_name=$(package)-$($(package)_version).tar.gz
 $(package)_sha256_hash=888fb0976ba61518276fe1eda988589c700a3f2a69d71089260d75562afd3687
-$(package)_patches=dont_leak_info.patch
+$(package)_patches=dont_leak_info.patch respect_mingw_cflags.patch
 
 define $(package)_set_vars
 $(package)_build_opts=CC="$($(package)_cc)"
 $(package)_build_opts_darwin=LIBTOOL="$($(package)_libtool)"
-$(package)_build_opts_mingw32=-f Makefile.mingw
+$(package)_build_opts_mingw32=-f Makefile.mingw CFLAGS="$($(package)_cflags) -D_WIN32_WINNT=0x0601"
```

Comment:
Does this actually work?

`Makefile.mingw` defines `_WIN32_WINNT` in CPPFLAGS, which it puts on the commandline after CFLAGS, so I would expect it to override whatever we put in CFLAGS here...

Reply:
> Does this actually work?

Yes.

`make -C depends/ HOST=x86_64-w64-mingw32 -j9 miniupnpc V=1`.

master (a9049dd296a47319038d8c34bcc1ca46c89dee9c):
```bash
x86_64-w64-mingw32-gcc -Wall -W -Wstrict-prototypes -Os -DNDEBUG -D_WIN32_WINNT=0X501 -DMINIUPNP_STATICLIB -c -o upnpcommands.o upnpcommands.c
```

This PR (1a332c78dbc78a8fd941289c21e662b10bd3b914):
```bash
x86_64-w64-mingw32-gcc -pipe -std=c11 -O2 -D_WIN32_WINNT=0x0601 -DMINIUPNP_STATICLIB -c -o upnpcommands.o upnpcommands.c
```

Reply:
Ah, apparently this part has changed in master miniupnpc. Maybe add a comment or assert-like check to ensure we don't bump it without fixing?

Reply:
I've added a comment for the next update.